### PR TITLE
Swift 3

### PR DESCRIPTION
**Pull request by Eke**
Migrated library to `Swift 3`, removed `Carthage`, added `Cocoapods`, updated scripts

### CODE REVIEW

File: MockingjayMatchers/MockingjayMatchers.swift [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-public func api(method: HTTPMethod, _ uri: String) -> (request: NSURLRequest) -> Bool {
+public func api(_ method: HTTPMethod, _ uri: String) -> (_ request: URLRequest) -> Bool {
     return { request in
         guard let headers = request.allHTTPHeaderFields,
-            accept = headers["Accept"],
-            acceptEncoding = headers["Accept-Encoding"],
-            contentType = headers["Content-Type"] where
+            let accept = headers["Accept"],
+            let acceptEncoding = headers["Accept-Encoding"],
+            let contentType = headers["Content-Type"] ,
```
Comment (nickvelloff) [inline MockingjayMatchers/MockingjayMatchers.swift]: extra comma

**Comment by nickvelloff** [comment]
Tiny fix, then ship this baby 🚢 !!

File: MockingjayMatchers/MockingjayMatchers.swift (hunk 1 above)
Comment (Eke) [reply MockingjayMatchers/MockingjayMatchers.swift]: hmmm, i think, this comma is needed there

Comment (hannesstruss) [reply MockingjayMatchers/MockingjayMatchers.swift]: Definitely extra space

Comment (Eke) [reply MockingjayMatchers/MockingjayMatchers.swift]: removed space

**Pull request merged**